src/client/components/classifier/lut.ts

```diff
@@ -19,11 +19,11 @@ export class Lut {
     this.size = size
   }
 
-  static of() {
+  static of(size: number) {
     return new Lut({
       atom: createAtom('Lut'),
-      data: new Uint8Array(2 ** (6 + 6 + 6)).fill(Classification.Unclassified),
```

Comment:
Our LUTs these days aren't the same size in all dimensions.
We normally have like 6x5x5 luts etc rather than square ones.
Not sure if that matters for this of function though

Reply:
And it seems this is gone already :P

Reply:
Wait nope, It's there I just somehow commented on the wrong side of the diff?

Reply:
haha 😄 I'll update it to take 3 args ea52ff1583b09b776a99ff7379c6d2653155c43c